app/src/main/java/msi/paria/trainingapp/widgets/buttons/RoundButton.kt

```diff
@@ -0,0 +1,27 @@
+package msi.paria.trainingapp.widgets.buttons
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+
+
+@Composable
+fun RoundButtonView(id: String, text: String, onSubmitButtonClick: () -> Unit) {
+    Button(modifier = Modifier
+        .fillMaxWidth()
+        .testTag(id),
```

Comment:
What is the point of passing the testTag?

Reply:
In future if we use multiple view in one screen, and if we use hard code tag, It's possible to use same test id. because of that our test case be failed.maybe it's better to get test if from outside of view